src/components/Articles.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { getArticles } from '../../utils/api';
+import { Link } from 'react-router-dom';
+
+const arts = [
+  {
+    author: 'grumpy19',
+    title: 'The Notorious MSG’s Unlikely Formula For Success',
+    article_id: 34,
+    topic: 'cooking',
+    created_at: '2020-11-22T11:13:00.000Z',
+    votes: 0,
+    article_img_url:
+      'https://images.pexels.com/photos/2403392/pexels-photo-2403392.jpeg?w=700&h=700',
+    comment_count: 11,
+  },
+  {
+    author: 'tickle122',
+    title: 'The battle for Node.js security has only begun',
+    article_id: 12,
+    topic: 'coding',
+    created_at: '2020-11-15T13:25:00.000Z',
+    votes: 0,
+    article_img_url:
+      'https://images.pexels.com/photos/10845119/pexels-photo-10845119.jpeg?w=700&h=700',
+    comment_count: 7,
+  },
+];
+
+export default function Articles() {
+  const [articles, setArticles] = useState(arts);
+  useEffect(() => {
+    getArticles().then((fetchedArticles) => {
+      setArticles(fetchedArticles);
+    });
+  }, []);
+  return (
+    <>
+      <h1>Articles</h1>
+      <div className='articles'>
+        {articles.map((article) => {
+          return (
+            <div className='article' key={article.article_id}>
```

Comment:
It could be more helpful for screen readers if this is a _ul_ or _ol_ with _li_ list items 💡